### PR TITLE
feat(replica_stats): adding protobuf changes

### PR DESCRIPTION
**Pull request by abhilashshetty04**

### CODE REVIEW

**Review by tiagolobocastro — APPROVED** [review]
Please squash the commits

File: apis/io-engine/protobuf/v1/stats.proto [hunk 1]
```diff
@@ -29,6 +31,17 @@ message NexusIoStatsResponse {
...
 }
 
+/// Replica IO stats response.
+message ReplicaIoStatsResponse {
+  repeated ReplicaIoStats stats = 1;
+}
+
+// Replica IO stats struct.
+message ReplicaIoStats {
+  optional string entity_id = 1; // Entity ID of the replica, If any.
```
Comment (dsharma-dc) [inline apis/io-engine/protobuf/v1/stats.proto:41]: If available, what is entity id w.r.t replica? The owner Volume?

**Review by dsharma-dc — APPROVED** [review]

File: apis/io-engine/protobuf/v1/stats.proto (hunk 1 above)
Comment (abhilashshetty04) [reply apis/io-engine/protobuf/v1/stats.proto:41]: Yes @dsharma-dc

**Review by avishnu — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
> 127 | impl StatsRpc for StatsService {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `get_replica_io_stats` in implementation

Again... please don't merge PR's on this repo when the mayastor/control PR is also about to be merged

**Pull request merged**